src/components/Pages/ProductPage/index.js

```diff
@@ -38,22 +40,48 @@ export const ProductPage = () => {
     })();
   }, [id]);
 
+  // console.log(state.count);
```

Comment:
Is this needed? 
Tip: before delivering, you can search for console.logs in your code and clean up the ones you don't need. This way, they rarely get past you